api/scanner/exif/exif_parser_external.go

```diff
@@ -103,13 +103,17 @@ func (p *externalExifParser) ParseExif(media_path string) (returnExif *models.Me
 	}
 
 	//Get time of photo
-	date, err := fileInfo.GetString("DateTimeOriginal")
-	if err == nil {
-		layout := "2006:01:02 15:04:05"
-		dateTime, err := time.Parse(layout, date)
+	createDateKeys := []string{"DateTimeOriginal", "CreateDate", "TrackCreateDate", "MediaCreateDate", "FileCreateDate", "ModifyDate", "TrackModifyDate", "MediaModifyDate", "FileModifyDate"}
+	for _, createDateKey := range createDateKeys {
+		date, err := fileInfo.GetString(createDateKey)
 		if err == nil {
-			found_exif = true
-			newExif.DateShot = &dateTime
+			layout := "2006:01:02 15:04:05"
```

Comment:
It does not solve the problem completely @viktorstrate 
I am still having the exact same problem using:
Version 2.3.13 ([446ef9a4])
Build-date 2022-07-26T18:26:27Z

Apprantly, the CreateDate can hold an UTC offset which makes the time.Parse fail:
https://go.dev/play/p/CrMGSIzoU1y

Exif:
```
ExifTool Version Number         : 11.70
File Name                       : 0000006.mov
Directory                       : .
File Size                       : 6.2 MB
File Modification Date/Time     : 2022:12:18 10:43:39+01:00
File Access Date/Time           : 2023:02:14 21:47:11+01:00
File Inode Change Date/Time     : 2023:02:14 21:48:10+01:00
File Permissions                : rw-r--r--
File Type                       : MOV
File Type Extension             : mov
MIME Type                       : video/quicktime
Major Brand                     : Apple QuickTime (.MOV/QT)
Minor Version                   : 0.0.0
Compatible Brands               : qt
Movie Data Size                 : 6496639
Movie Data Offset               : 36
Movie Header Version            : 0
Create Date                     : 2022:12:18 09:43:39
Modify Date                     : 2022:12:18 09:43:39
Time Scale                      : 600
Duration                        : 17.31 s
Preferred Rate                  : 1
Preferred Volume                : 100.00%
Preview Time                    : 0 s
Preview Duration                : 0 s
Poster Time                     : 0 s
Selection Time                  : 0 s
Selection Duration              : 0 s
Current Time                    : 0 s
Next Track ID                   : 3
Track Header Version            : 0
Track Create Date               : 2022:12:18 09:43:39
Track Modify Date               : 2022:12:18 09:43:39
Track ID                        : 1
Track Duration                  : 17.31 s
Track Layer                     : 0
Track Volume                    : 0.00%
Image Width                     : 480
Image Height                    : 848
Clean Aperture Dimensions       : 480x848
Production Aperture Dimensions  : 480x848
Encoded Pixels Dimensions       : 480x848
Graphics Mode                   : ditherCopy
Op Color                        : 32768 32768 32768
Compressor ID                   : avc1
Source Image Width              : 480
Source Image Height             : 848
X Resolution                    : 72
Y Resolution                    : 72
Compressor Name                 : 'avc1'
Bit Depth                       : 24
Video Frame Rate                : 29.919
Matrix Structure                : 1 0 0 0 1 0 0 0 1
Media Header Version            : 0
Media Create Date               : 2022:12:18 09:43:39
Media Modify Date               : 2022:12:18 09:43:39
Media Time Scale                : 44100
Media Duration                  : 17.35 s
Media Language Code             : und
Balance                         : 0
Handler Class                   : Data Handler
Handler Vendor ID               : Apple
Handler Description             : Core Media Data Handler
Audio Format                    : mp4a
Audio Channels                  : 2
Audio Bits Per Sample           : 16
Audio Sample Rate               : 44100
Purchase File Format            : mp4a
Handler Type                    : Metadata Tags
Creation Date                   : 2022:08:25 16:25:20+02:00
Image Size                      : 480x848
Megapixels                      : 0.407
Avg Bitrate                     : 3 Mbps
Rotation                        : 0
```

CreateDate would be the wanted date but it is using a different one, pointing to 2022:12:18 instead of 2022:08:25.

Reply:
Made a pull_request -> https://github.com/photoview/photoview/pull/802